client/src/components/course-view/OngoingInstanceInfo.js

```diff
@@ -11,10 +11,8 @@ import LightLabelBoldValue from '../typography/LightLabelBoldValue';
 import textFormatServices from '../../services/textFormat';
 import useAuth from '../../hooks/useAuth';
 
-const mockInstitution = 'Aalto University';
-
 const OngoingInstanceInfo = ({ info }) => {
```

Comment:
Should the title here be changed from 'Ongoing Instance' to 'Instance Details' (or something similar) since now details are shown here that aren't only about the ongoing instance but instead you can choose previous instances?

Reply:
Agreed